docs/guides/03-Styling-your-visualization-using-Expressions.md

```diff
@@ -0,0 +1,198 @@
+## Styling your visualization using Expressions
+
+CARTO VL styling is based on the concept of visualizations. A visualization is a set of styling properties and variables that are assigned expressions. Visualizations allow you to control everything from the color of features to the available properties in interactivity events.
+
+### Styling properties
+Visualizations have a fixed set of styling properties like `color` or `width` that adapt automatically to the geometry type:
+
+<div class="example-map">
+    <iframe
+        id="population-density-basic"
+        src="/developers/carto-vl/examples/maps/guides/viz-properties/geom-types.html"
+        width="100%"
+        height="1000"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+    <a href="/developers/carto-vl/examples/#example-geometry-types">View my source code!</a>
+</div>
+
+The complete list of CARTO VL styling properties and their semantics can be seen [here](https://carto.com/developers/carto-vl/reference/#vizspec).
```

Comment:
I think it's better to use just '/developers/carto-vl/reference/#vizspec' for greater flexibility